documentation/modules/configuring/con-config-mirrormaker2-replication.adoc

```diff
@@ -81,6 +85,28 @@ Both topics enable replication to be fully restored from the correct offset posi
 
 MirrorMaker 2.0 uses its `MirrorCheckpointConnector` to emit _checkpoints_ for offset tracking.
 
+== Synchronizing consumer group offsets
+
+The `__consumer_offsets` topic stores information on committed offsets, according to consumer group.
+Offset synchronization periodically transfers the consumer offsets for the consumer groups of a source cluster into the consumer offsets topic of a target cluster.
+
+Offset synchronization is particularly useful in an _active/passive_ configuration.
+Should the active cluster go down, consumer applications can switch to the passive (standby) cluster and pick up from the last offset position.
+
+To use topic offset synchronization:
+
+* Enable the synchronization by adding `sync.group.offsets.enabled` to the checkpoint connector configuration of the `KafkaMirrorMaker2` resource,
+and setting the property to `true`. Synchronization is disabled by default.
+* Add the `IdentityReplicationPolicy` to the source connector configuration so that topics in the target cluster retain their original names.
+
+For topic offset synchronization to work, consumer groups in the source cluster cannot be active in the target cluster.
+Consequently, if you are using an _active/active_ configuration that does not support or require offset synchronization, do not enable topic offset synchronization.
+
+If enabled, the synchronization of offsets from the source cluster is made periodically.
+You can change the frequency by adding `sync.group.offsets.interval.seconds` and `emit.checkpoints.interval.seconds` to the checkpoint connector configuration.
+The properties specify the frequency in seconds that the consumer group offsets are synchronized, and the frequency of checkpoints emitted for offset tracking.
+The default for both properties is 60 seconds.
```

Comment:
Should this mention also the `refresh.groups.interval.seconds` interval? (default 10 minutes)

Reply:
We should also be explicit that because this is time based it means that consumers which switch to the passive on failover will process some (hopefully small) number of duplicates.

Reply:
Updated to:

> If enabled, the synchronization of offsets from the source cluster is made periodically.
> You can change the frequency by adding `sync.group.offsets.interval.seconds` and `emit.checkpoints.interval.seconds` to the checkpoint connector configuration.
> The properties specify the frequency in seconds that the consumer group offsets are synchronized, and the frequency of checkpoints emitted for offset tracking.
> The default for both properties is 60 seconds.
> You might also want to change the frequency of checks for new consumer groups using the `refresh.groups.interval.seconds` property, which is performed every 10 minutes by default.
> 
> Because the synchronization is time-based, any switchover by consumers to a passive cluster will likely result in some duplication of messages.

Also added `refresh.groups.interval.seconds` to example config